spec/fixtures/generate_external_certs.sh

```diff
@@ -0,0 +1,322 @@
+#!/bin/bash
+
+# This script re-generates all of the keys and certs within the external_certs directory.
+# The files there are used by the spec/features/settings/external_cert_feature_spec test
+
+function init() {
+  ROOT_DIR=/tmp/ca_gen
+  CSR_DIR=$ROOT_DIR/csr
+  DIR_CERTS=$ROOT_DIR/certs
+  DIR_CONF=$ROOT_DIR/conf
+  DIR_KEYS=$ROOT_DIR/keys
+  DIR_CADATA=$ROOT_DIR/cadata
+  
+  mkdir -p $CSR_DIR
+  mkdir -p $DIR_CERTS
+  mkdir -p $DIR_CONF
+  mkdir -p $DIR_KEYS
+  mkdir -p $DIR_CADATA
+}
+
+function cleanup() {
+  rm -rf $ROOT_DIR
```

Comment:
I like sticking a `-v` on the cleanup removal for log purposes, but that's up to you.
```suggestion
  rm -vrf $ROOT_DIR
```

Reply:
Makes nice verbose output on the `mkdir` calls, too. ;)

Reply:
Sounds good to me. Adding.